src/main/java/utils/StringToDate.java

```diff
@@ -0,0 +1,141 @@
+package utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * This class returns literal dates converted to date format.
+ */
+public class StringToDate {
+
+    SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
+
+    /**
+     * Converts literal date String to a date format.
+     *
+     * @param stringToConvert a literal date String.
+     * @return a date format.
+     */
+    public Date convert(final String stringToConvert) {
+        if (stringToConvert == null) {
+            throw new IllegalArgumentException("Value is null");
+        }
+        if (stringToConvert.equals("")) {
+            throw new IllegalArgumentException("Value is empty");
+        }
+
+        if (stringToConvert.matches("TODAY|TOMORROW|YESTERDAY")) {
+            return dayName(stringToConvert);
+        }
+
+        if (stringToConvert.contains("ago") || stringToConvert.contains("from now")) {
```

Comment:
Maybe is necessary a method to validate the string, in case String contain other words.

Reply:
In this case no. If the string doesn't enter any of the if's, at the end an exception will be thrown